ui/utils/custom-search.js

```diff
@@ -123,7 +123,12 @@ const SearchBar = ({ onSearch, placeholder, expanded, setExpanded, value = '' })
       ) : (
         <CustomTooltip title="Search">
           <IconButton onClick={handleSearchIconClick}>
-            <SearchIcon fill={theme.palette.icon.secondary} height={'1.5rem'} width={'1.5rem'} />
+            <SearchIcon
+              data-testid="search-icon"
```

Comment:
Also would we be able to lift this to a higher level on the component? If its not we may need to add a modifier to this as this toggles between expanded/collpased states